draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1031,21 +1048,21 @@ document's "site for cookies".
 
 Shared workers may be bound to multiple documents at once. As it is quite
 possible for those documents to have distinct "site for cookie" values, the
-worker's "site for cookies" will be the empty string in cases where the values
-diverge, and the shared value in cases where the values agree.
+worker's "site for cookies" will be a globally unique identifier in cases
+where the values diverge, and the shared value in cases where the values agree.
```

Comment:
(Hopefully we can double-key these so this oddity no longer matters. As things stand, this feature is... kinda weird.)